drivers/sensor/pni/rm3100/rm3100_stream.c

```diff
@@ -29,6 +30,12 @@ static void rm3100_complete_result(struct rtio *ctx, const struct rtio_sqe *sqe,
 				     data->stream.settings.enabled.drdy);
 	edata->header.channels = 0;
 
+	if (data->settings.odr == RM3100_DT_ODR_600) {
+		edata->header.cycle_count = RM3100_CYCLE_COUNT_HIGH_ODR;
+	} else {
+		edata->header.cycle_count = RM3100_CYCLE_COUNT_DEFAULT;
+	}
```

Comment:
I see this was present in one-shot mode through `rm3100_encode`. I suggest to use in streaming as well.